mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -27,6 +27,7 @@ import styles from "src/logic/wayPage/reportsTable/WayColumns.module.scss";
 
 const DEFAULT_SUMMARY_TIME = 0;
 const columnHelper = createColumnHelper<DayReport>();
+const ONE = 1;
```

Comment:
One apple? Or maybe one boy? Could we reuseit?

Reply:
Don;t know hor to rename

Reply:
think about it

Reply:
Created function getListNumberByIndex but I also need to create const for 1 (because it's a magic number)